core-tests/src/test/java/org/mule/runtime/core/internal/processor/AsyncDelegateMessageProcessorTestCase.java

```diff
@@ -145,6 +150,26 @@ public void processWithTx() throws Exception {
     }
   }
 
+  @Test
+  @Ignore("Does this case actually make sense?")
```

Comment:
Add a comment indicating why this was left (subscriber context is not propagated in the case of async)